helper/identity/templating.go

```diff
@@ -86,12 +93,25 @@ func performTemplating(ns *namespace.Namespace, input string, entity *Entity, gr
 				return "", ErrTemplateValueNotFound
 			}
 			return alias.Name, nil
-		case strings.HasPrefix(trimmed, "metadata."):
-			val, ok := alias.Metadata[strings.TrimPrefix(trimmed, "metadata.")]
-			if !ok {
-				return "", ErrTemplateValueNotFound
+		case strings.HasPrefix(trimmed, "metadata.") || trimmed == "metadata":
+			split := strings.SplitN(trimmed, ".", 2)
+
+			switch len(split) {
+			case 1:
+				// all metadata as json, without outer braces
```

Comment:
To me it seems to make more sense to leave the outer braces and trim them elsewhere if needed because now we're returning a string that is for use in json but...isn't.

Reply:
It is definitely a little weird, though arguably more consistent with strings. The string parameters are currently returning `foo`, not `"foo"`, and the template provides the wrapping quotes or braces.  But, I don't love it. Lemme see what I can do here...

Reply:
Redid this so in JSON mode, you'll get back strings, objects or lists.